config/locales/client.ar.yml

```diff
@@ -9,7 +9,7 @@ ar:
     category:
       sort_options:
         votes: "الأصوات"
-    voting:
+    topic_voting:
```

Comment:
@gschlager (I'm sorry that the translation process is still a blackbox for me 😄) Is this mass-rename of keys a-ok with crowdin?

Reply:
Yes, that's okay. Most of our Crowdin workflows contain a workflow step that uses Translation Memory to pretranslate new strings when there's a 100% match with a previous source string.